### ISSUE TITLE
Crashes after upgrading from 8.3.11 to 8.4.4

### ISSUE THREAD

**Issue by lpil** [ismay/stylelint-no-unsupported-browser-features#224]
Hello! We upgraded today and got this crash

```
TypeError: opts.node.rangeBy is not a function
    at new Warning (/Users/louis/src/nomio/nomio/node_modules/stylelint-no-unsupported-browser-features/node_modules/postcss/lib/warning.js:9:29)
    at Result.warn (/Users/louis/src/nomio/nomio/node_modules/stylelint-no-unsupported-browser-features/node_modules/postcss/lib/result.js:26:19)
    at /Users/louis/src/nomio/nomio/node_modules/doiuse/lib/doiuse.js:65:16
    at Detector.decl (/Users/louis/src/nomio/nomio/node_modules/doiuse/lib/detect-feature-use.js:70:11)
    at /Users/louis/src/nomio/nomio/node_modules/doiuse/lib/detect-feature-use.js:155:16
    at Rule.each (/Users/louis/src/nomio/nomio/node_modules/postcss/lib/container.js:105:16)
    at Detector.node (/Users/louis/src/nomio/nomio/node_modules/doiuse/lib/detect-feature-use.js:148:10)
    at Detector.rule (/Users/louis/src/nomio/nomio/node_modules/doiuse/lib/detect-feature-use.js:90:10)
    at /Users/louis/src/nomio/nomio/node_modules/doiuse/lib/detect-feature-use.js:151:16
    at Rule.each (/Users/louis/src/nomio/nomio/node_modules/postcss/lib/container.js:105:16)
```

**Comment by ismay** [comment]
Hi @lpil, thanks for reporting this. I can reproduce it, but don't know what's causing it yet. I assume the unit tests didn't catch this as the installed deps are cached (though it's an assumption that the current unit tests would catch this).

The first thing to do would be to figure out where it broke to see what's causing it exactly. I see the original title had a version range, so assuming it broke inbetween those two versions a running git bisect on this repo would probably help narrow down where it broke.

I don't have a lot of time to look into this at the moment, but if you'd like to help out then if you could look into where it broke exactly that would be great! To simplify testing during the bisect you could see whether running a clean install and then this repo's unit tests can reproduce the bug. If that's the case then all you'd have to do to test is `npm ci && npm test`.

**Comment by PxlBuzzard** [comment]
I think this will end up being the issue that fixes this https://github.com/stylelint/stylelint/issues/5766
